homeassistant/components/nut/config_flow.py

```diff
@@ -27,17 +26,39 @@
 
 DATA_SCHEMA = vol.Schema(
     {
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): str,
-        vol.Required(CONF_RESOURCES): cv.multi_select(SENSOR_DICT),
         vol.Optional(CONF_HOST, default=DEFAULT_HOST): str,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): int,
-        vol.Optional(CONF_ALIAS): str,
         vol.Optional(CONF_USERNAME): str,
         vol.Optional(CONF_PASSWORD): str,
     }
 )
 
 
+def _resource_schema(available_resources, selected_resources):
+    """Resource selection schema."""
+
+    known_available_resources = {
+        sensor_id: SENSOR_TYPES[sensor_id][0]
+        for sensor_id in SENSOR_TYPES
+        if sensor_id in available_resources
```

Comment:
```suggestion
        sensor_id: sensor[0]
        for sensor_id, sensor in SENSOR_TYPES.items()
        if sensor_id in available_resources
```

Reply:
Was this missed when committing?

Reply:
Yes, thanks.  github out of sync and rejected it without me noticing.